Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM python:3.12-slim
+
+# Prevents generating .pyc files in the container
```

Comment:
question: I would think that you want to pre-generate `.pyc` files in the container, since it's not possible for the source code to change?

Reply:
thats a good point. i'll make that change.